package.json

```diff
@@ -42,7 +42,13 @@
     "slate-react": "^0.21.20",
     "styled-components": "^5.2.0",
     "styled-system": "^5.0.6",
-    "unified": "^8.3.2"
+    "unified": "^8.3.2",
+    "node-forge": "^0.10.0",
+    "node-fetch": "^2.6.1",
+    "yargs-parser": "^13.1.2",
+    "http-proxy": "^1.18.1",
+    "dot-prop": "^4.2.1",
+    "serialize-javascript": "^3.1.0"
```

Comment:
what do all of these do?

Reply:
Fix vulnerabilities

Reply:
I keep getting github dependabot notifications